docs/curation-review-tut.md

```diff
@@ -0,0 +1,417 @@
+# Cellxgene Curation Tutorial
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png) --->
+
+<img align="right" src="https://user-images.githubusercontent.com/25663501/111377133-b07a4c80-8676-11eb-8eb8-07ca4d7a77e9.png">
+
+
+
+
+<!--- ## Overview --->
+<!--- purpose: Motivate submission to the data portal give context --->
+
+[Cellxgene's publishing platform](https://cellxgene.cziscience.com/) and [interactive single cell data explorer](https://github.com/chanzuckerberg/cellxgene) is a system which is optimized for easy access, interrogation and sharing/reuse of single cell data. In order to meet these standards and to facilitate easy navigation/aggregation among the growing number of single cell datasets, the cellxgene publishing platform currently accepts curated anndata objects adhering to a [comprehensive data schema](corpora_schema.md). In this tutorial, you will gain all the essential information for doing the following: 
+- curating one of your own single cell datasets in a streamlined manner
+- using CZI curation tools to perform most of the curation work
+- uploading data in the cellxgene data portal
+- navigating the portal and ensuring public or private access to your datasets
+- basic data exploration in the cellxgene explorer view
+
+Beyond this we hope that you see the value of using this portal as a publishing platform and also welcome any comments that pertain to improving the portal and curation experience. You can contact us via [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com). You can also check out the following links for more information on the cellxgene ecosytem:
+
+- [cellxgene data portal](https://cellxgene.cziscience.com/)
+- [cellxgene explorer github](https://github.com/chanzuckerberg/cellxgene)
+- [cellxgene data portal github](https://github.com/chanzuckerberg/corpora-data-portal)
+- [corpora schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/docs/corpora_schema.md)
+- [schema guide](https://github.com/chanzuckerberg/single-cell-curation/blob/main/docs/schema_guide.md)
+
+<details>
+<summary> Table of contents </summary>
+
+- [Quick start](#quick-start)
+- [Why contribute?](#why-contribute)
+- [Basic data requirements](#basic-data-requirements)
+- [Anndata object requirements](#anndata-object-requirements)
+- [Schema definition](#schema-definition)
+- [Cellxgene curation tools](#cellxgene-curation-tools)
+
+</details>
+
+<br/>
+
+---
+
+## Quick start
+
+<br/>
+
+If you already know everything you need to know, then give curation a shot with this [quick start guide](https://github.com/chanzuckerberg/single-cell-curation#quick-start). Otherwise, continue reading!
+
+---
+
+## Why contribute?
+
```

Comment:
Who is the audience for this section? 

It feels targeted towards attracting individual scientists to publish.
Curators are funded and should be more focused on the process rather than requiring motivation.

Reply:
@pablo-gar and @MaximilianLombardo field an increasing number of requests from external users who we identify as having high quality datasets, but which we lack the time to curate. The hope is they can be shown this document and it will provide them with enough information that Pablo or Max won't need to devote time to training them on our tools.

Reply:
Per Jason's comments in the gene sets doc, this is also pending information about the use of gene sets in cellxgene.